docs/DeveloperGuide.md

```diff
@@ -133,6 +133,44 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Alias feature
+The `Alias` feature allows users to define a shortcut for a longer command that is often used. The longer command can then be executed by entering the `Alias` instead of the full or partial command.
```

Comment:
Perhaps 

> can then be executed by entering the `Alias` instead of the full or partial command.

should instead be

> can then be executed by entering the alias instead of the full or partial command.

because the user is not typing in `Alias` but rather the alias that they specified. It's a small semantic point.

Reply:
Done!